### ISSUE TITLE
Remove ShellJS or migrate to local imports

### ISSUE THREAD

**Issue by mjalkio** [serverless/serverless#4464]
# This is a Bug Report

## Description

I am using a tool called Snyk which checks dependencies for security vulnerabilities. Currently, serverless is considered a vulnerable project due to one of its dependencies, [ShellJS](https://github.com/shelljs/shelljs).

The vulnerability in ShellJS is around the `shell.exec()` function, and I don't believe that Serverless uses this function. Therefore, it's not a vulnerability I need to worry about. However, because ShellJS is imported globally: https://github.com/serverless/serverless/blob/464d0b22b526132f1c773dfede6cac57de6cc8e9/lib/Serverless.js#L3 I can't actually figure out where it's used at all.

Can anyone provide insight on where functions from shelljs are used in Serverless and whether it can be removed as a dependency? If it is used, my recommendation is to move to using local imports for ShellJS, which is also what [they recommend](https://github.com/shelljs/shelljs#global-vs-local).

Before posting here I asked about ShellJS on the [Serverless forum](https://forum.serverless.com/t/shelljs-use-in-serverless/2945) but didn't receive a response.

## Additional Data

* ***Serverless Framework Version you're using***: N/A
* ***Operating System***: N/A
* ***Stack Trace***: N/A
* ***Provider Error messages***: N/A

**Comment by RafalWilinski** [comment]
Thanks reporting an issue @mjalkio! 💯 

I've just removed this line locally to see what's going to happen, here are results:
- Tests are not failing
- Basic functionality works, not sure about more complex scenarios

Seems that this was added to make this line: https://github.com/serverless/serverless/commit/ab9d8a7c1f3928bdfe1e46055794fdbb335680ee#diff-d734f22324008b6cec2bfb478d6e74a3R259 work. It seems that we're safe to remove this, however, we should ensure that it's safe first.
